source/help/getting-started/configuring-notifications.rst

```diff
@@ -0,0 +1,105 @@
+Configuring Notifications
+=========================
+
+Notifications in Mattermost alert you to unread messages and mentions.
+
+Unreads and Mentions
+----------------------------------------
+
+Unread Messages Indicator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The name of a channel in the left-hand sidebar shows as **bold** when
+there are unread messages in the channel. Clicking on the channel
+removes the bold indicator, and brings you to the earliest unread
+message in the channel.
+
+You can choose not to show unread indicators in a channel by clicking
+**Channel Menu** > **Notification Preferences** > **Mark Channel
+Unread** and selecting **Only for mentions**. This will only notify you if a mention is triggered on your name or a
+keyword for which you're listening.
+
+Mentions Indicator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Unread mentions or `your name and selected
+keywords <https://docs.mattermost.com/help/settings/account-settings.html#words-that-trigger-mentions>`__
+in a channel are indicated by bold text and a mention counter next to
+the channel name in the left-hand sidebar.
+
+Clicking the channel name removes the bolding and mention count. You can
+review your recent mentions by clicking **@** next to the search box
+at the top of the screen.
+
+Learn more about `mentioning
+teammates <http://docs.mattermost.com/help/messaging/mentioning-teammates.html>`__.
+
+Email Notifications
+-------------------------------------
+
+Emails are sent for any mentions you receive if you have Mattermost
+closed or have not had any browser activity for a short time.
+
+-  Turn email notifications On or Off in **Account Settings** >
+   **Notifications** > **Email Notifications**.
+-  Configure the email address where notifications are sent in **Account
+   Settings** > **General** > **Email**
+
+Desktop Notifications
+-------------------------------------
+
+These are browser notifications that appear in the corner of your main
+monitor. By default, these notifications are sent for all activity while
+Mattermost is open. Desktop notifications are available on Edge, Firefox,
+Safari, Chrome and `Mattermost Desktop Apps <https://about.mattermost.com/download/#mattermostApps>`_.
+
+-  Configure when desktop notifications are sent from **Account
+   Settings** > **Notifications** > **Desktop Notifications** > **Send
+   Desktop Notifications**.
+-  Configure channel specific desktop notifications from **Channel
+   Menu** > **Notification Preferences** > **Send Desktop
+   Notifications**. By default, all channels use the global setting
+   configured in *Account Settings*.
```

Comment:
Yes we decided that Italics are for windows or sections that guide the user but not for clicks.